requirements.production.txt

```diff
@@ -0,0 +1,5 @@
+Flask
+flask-basicauth
+uwsgi
```

Comment:
Only `uwsgi` should be in requirements.production.txt, requirements should not be duplicated.